src/components/screens/login/networkSelector/networkSelector.js

```diff
@@ -76,7 +76,7 @@ const NetworkSelector = ({
 
   // eslint-disable-next-line max-statements
   const validateCorrectNode = async (networkName) => {
-    const networkToSet = getNetwork(networkName);
+    const networkToSet = networks[networkName];
```

Comment:
This was using the getNetwork function from the component itself, not the one you removed.

Reply:
oh, Thanks.